src/engraving/dom/utils.cpp

```diff
@@ -1334,4 +1335,16 @@ String formatUniqueExcerptName(const String& baseName, const StringList& allExce
 
     return result;
 }
+
+bool isFirstSystemKeySig(const KeySig* ks)
+{
+    if (!ks) {
+        return false;
+    }
+    const System* sys = ks->measure()->system();
+    if (!sys) {
+        return false;
+    }
+    return ks->tick() == sys->firstMeasure()->tick();
```

Comment:
I think it's preferable to check equality of the `.tick()` fractions directly instead of via `.ticks()`, because the conversion to integers is a bit more expensive and a lossy for fractions that are not multiples of 1/480 (although that won't likely be a problem for key signatures)